core/src/test/java/org/frankframework/senders/JavascriptSenderTest.java

```diff
@@ -27,6 +31,11 @@ class JavascriptSenderTest extends SenderTestBase<JavascriptSender> {
 
 	private Message dummyInput;
 
+	@BeforeAll
+	public static void setup() {
+		assumeFalse(TestAssertions.isTestRunningOnARM());
```

Comment:
```suggestion
		// Since it's not possible to conditionally check the EnumSource here, just disable all for now...
		assumeFalse(TestAssertions.isTestRunningOnARM(), "uses J2V8 which does not work on ARM");
```